tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,72 +0,0 @@
-const expect = require('chai').expect
-const rskUtils = require('../lib/rsk-utils');
-const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
-const { getBtcClient } = require('../lib/btc-client-provider');
-const { sendPegin, ensurePeginIsRegistered, sendTxToBridge } = require('../lib/2wp-utils');
-const { getBridge } = require('../lib/precompiled-abi-forks-util');
-const { getDerivedRSKAddressInformation } = require('@rsksmart/btc-rsk-derivation');
-const btcEthUnitConverter = require('@rsksmart/btc-eth-unit-converter');
-const whitelistingAssertions = require('../lib/assertions/whitelisting');
-
-describe('Transfer BTC to RBTC before papyrus200', function() {
-
-  let rskTxHelpers;
-  let btcTxHelper;
-  let rskTxHelper;
-
-  const PEGIN_VALUE_IN_BTC = 3;
-  const PEGOUT_VALUE_IN_RBTC = PEGIN_VALUE_IN_BTC / 3;
-  const RSK_TX_FEE_IN_RBTC = 0.001;
-  
-  before(async () => {
-      rskTxHelpers = getRskTransactionHelpers();
-      rskTxHelper = rskTxHelpers[0];
-      btcTxHelper = getBtcClient();
-  });
-
-
-  it('should do a multiple pegouts from the same rsk address after making a pegin', async () => {
```

Comment:
This test is not doing much really. Just multiple pegouts from the same rsk address at different times. If one rsk address can make a pegout once, it can do it again. I think there's no need for this test.